src/test/java/com/puppycrawl/tools/checkstyle/checks/HexLiteralCaseCheckTest.java

```diff
@@ -0,0 +1,72 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code and other text files for adherence to a set of rules.
+// Copyright (C) 2001-2025 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks;
+
+import static com.google.common.truth.Truth.assertWithMessage;
+import static com.puppycrawl.tools.checkstyle.checks.HexLiteralCaseCheck.MSG_KEY;
+
+import org.junit.jupiter.api.Test;
+
+import com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+public class HexLiteralCaseCheckTest
+        extends AbstractModuleTestSupport {
+
+    @Override
+    protected String getPackageLocation() {
+        return "com/puppycrawl/tools/checkstyle/checks/hexliteralcase";
+    }
+
+    @Test
+    public void testGetRequiredTokens() {
+        final HexLiteralCaseCheck check = new HexLiteralCaseCheck();
+        final int[] expectedTokens = {TokenTypes.NUM_LONG, TokenTypes.NUM_INT};
+        assertWithMessage("Default required tokens are valid")
+            .that(expectedTokens)
+            .isEqualTo(check.getRequiredTokens());
+    }
+
+    @Test
+    public void testCheck()
+            throws Exception {
```

Comment:
Move this method to the end of class , to group together technical coverage of token method